corgi/tasks/management/commands/loadpulpdata.py

```diff
@@ -0,0 +1,68 @@
+import sys
+
+from django.core.management.base import BaseCommand, CommandParser
+
+from corgi.core.models import ProductComponentRelation, ProductStream
+from corgi.tasks.brew import fetch_modular_builds, slow_fetch_brew_build
+from corgi.tasks.pulp import logger
+
+
+class Command(BaseCommand):
+
+    help = "Fetch builds for cdn repos"
+
+    def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument(
+            "stream_name",
+            type=str,
+            help="Fetch builds using cdn_repo for variants configured in stream",
+        )
+        parser.add_argument(
+            "-i",
+            "--inline",
+            action="store_true",
+            help="Schedule build for ingestion inline (not in celery)",
+        )
+        parser.add_argument(
+            "-f",
+            "--force",
+            action="store_true",
+            help="Force ingestion even if the Build exists",
+        )
+
+    def handle(self, *args, **options) -> None:
+        if options["stream_name"]:
+            stream_name = options["stream_name"]
+            self.stdout.write(self.style.NOTICE(f"Fetching builds for stream: {stream_name}"))
+            build_ids = get_builds_by_cdn_repo(stream_name)
+
+        if not build_ids:
+            self.stderr.write(self.style.ERROR("No build IDs found"))
+            sys.exit(1)
+
+        self.stdout.write(
+            self.style.SUCCESS(
+                f"Fetching component data for builds: {', '.join(map(str, build_ids))}"
+            )
+        )
+
+        for build_id in build_ids:
+            build_id = int(build_id)
+            if options["inline"]:
+                slow_fetch_brew_build(build_id, force_process=options["force"])
+            else:
+                slow_fetch_brew_build.delay(build_id, force_process=options["force"])
```

Comment:
This passes a string build_id from get_builds_by_cdn_repo() / fetch_modular_builds(), but slow_fetch_brew_build() expects an int build_id.
```suggestion
            if options["inline"]:
                slow_fetch_brew_build(int(build_id), force_process=options["force"])
            else:
                slow_fetch_brew_build.delay(int(build_id), force_process=options["force"])
```

Reply:
this is all fixed in my uncommitted local changes too.